plugins/spacelift/api_key_test.go

```diff
@@ -0,0 +1,62 @@
+package spacelift
+
+import (
+	"testing"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+	
+func TestAPIKeyProvisioner(t *testing.T) {
+	plugintest.TestProvisioner(t, APIKey().DefaultProvisioner, map[string]plugintest.ProvisionCase{
+		"default": {
+			ItemFields: map[sdk.FieldName]string{
+				fieldname.Endpoint: "https://end.point",
+				fieldname.APIKeyID: "abc123",
+				fieldname.APIKeySecret: "def456",
+			},
+			ExpectedOutput: sdk.ProvisionOutput{
+				Environment: map[string]string{
+					"SPACELIFT_API_KEY_ENDPOINT": "https://end.point",
+					"SPACELIFT_API_KEY_ID": "abc123",
+					"SPACELIFT_API_KEY_SECRET": "def456",
+				},
+			},
+		},
+	})
+}
+
+func TestAPIKeyImporter(t *testing.T) {
+	plugintest.TestImporter(t, APIKey().Importer, map[string]plugintest.ImportCase{
+		"environment": {
+			Environment: map[string]string{
+				"SPACELIFT_API_KEY_ENDPOINT": "xhdw",
+				"SPACELIFT_API_KEY_ID": "abc123",
```

Comment:
When the lengths are set for the field names, you can run `make spacelift/example-secrets` to generate example secrets and use those for the tests. Here's what I generated for now: API Key `KKJNW2KPSTGJYQIBYWBEXAMPLE` and API Secret `5mtg0fl6rbta1l40vvbll5q94edy0uum1bkj09yyjang4qoutyrrnxx0oexample`.

Reply:
@arunsathiya is there any way to exclude `Endpoint` when generating these secrets? I get this error:
```
panic: runtime error: makeslice: len out of range
```
...when trying to run `make spacelift/example-secrets` with missing `Length` on the endpoint field.